release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3776,4 +3777,23 @@ $ oc adm release info 4.10.50 --pullspecs
 [id="ocp-4-10-50-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-51"]
+=== RHSA-2023:0561 - {product-title} 4.10.51 bug fix and security update
+
+Issued: 2023-02-08
+
+{product-title} release 4.10.51, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:0561[RHSA-2023:0561] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:0560[RHSA-2023:0560] advisory.
```

Comment:
Suggesting some nips and tucks. 

```suggestion
{product-title} release 4.10.51, which includes security updates, is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:0561[RHSA-2023:0561] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:0560[RHSA-2023:0560] advisory.
```

Reply:
Thanks, @aldiazRH . I took the stock text for granted. Thanks for bringing these minimalism suggestions to my attention. I've updated the text. Please let me know your throughts.

Reply:
Looks great, @dfitzmau , thanks! :+1: